mod_auth/models.py

```diff
@@ -23,35 +23,70 @@ class User(Base):
     role = Column(Role.db_type())
 
     def __init__(self, name, role=Role.user, email=None, password=''):
+	'''
+	Parametrized constructor for the User model
+	'''
         self.name = name
         self.email = email
         self.password = password
         self.role = role
 
     def __repr__(self):
+	'''
+	Representation function
+	Represent a User Model by its 'name' Field.
+	'''
         return '<User %r>' % self.name
 
     @staticmethod
     def generate_hash(password):
+	'''
+	Generates a Hash value for a password
+	:param password: The password to be hashed
+	:type password: str
```

Comment:
:return and :rtype are missing. They provide some help when using an IDE. Please add for all methods that return something.